test/testcafe/framework/page-objects/EnrollProfileViewPageObject.js

```diff
@@ -45,4 +45,23 @@ export default class EnrollProfileViewPageObject extends BasePageObject {
     return Selector(CUSTOM_IDP_BUTTON).nth(index).textContent;
   }
 
+  signUpButtonExists() {
+    return this.form.getButton('Sign Up').exists;
```

Comment:
Do we need to test this in a different language? If so, this is not going to work as is. The css selectors we had before would have worked, but if we are using the text now we would need that text to be localized in the test

Reply:
Currently we don't test different languages in the testcafe spec test, we went with the semantic approach to select buttons because of a few reasons, one of them being it made the transition into v3 (from v2) much simpler, where elements and selectors did not exactly match, we could select based on text and ensure the functionality matched. 

We did discuss being able to test different languages in SIW OH, but we were thinking it would still be this approach, however, we would maintain some sort of map that can map the expected text to a language for a given test suite. We have not discussed it since then but going forward I believe we want to stick with the semantic approach for selecting elements.